src/view/WinningNumbersView.js

```diff
@@ -0,0 +1,48 @@
+export default class WinningNumbersView {
+  constructor() {
+    this.$winningLottoSection = document.querySelector('#winning-lotto-section');
+    this.$winningLottoForm = document.querySelector('#winning-lotto-form');
+    this.$winningNumbersInputs = document.querySelectorAll('.winning-numbers-inputs');
+    this.$bonusNumberInput = document.querySelector('#bonus-number-input');
+    this.$resultButton = document.querySelector('#check-result-button');
+    this.connectEvents();
+  }
+
+  connectEvents() {
+    this.$winningLottoForm.addEventListener('submit', (e) => this.handleSubmit(e));
+  }
+
+  handleSubmit(e) {
+    e.preventDefault();
+
+    const winningLotto = {
+      winningNumbers: Array.from(this.$winningNumbersInputs).map((input) => input.valueAsNumber),
+      bonusNumber: this.$bonusNumberInput.valueAsNumber,
+    };
+
+    const event = new CustomEvent('check', { detail: winningLotto });
+
+    this.$winningLottoForm.dispatchEvent(event);
+  }
+
+  showWinningNumbers() {
+    this.$winningLottoSection.classList.replace('hidden', 'show');
+  }
+
+  hideWinningNumbers() {
+    this.$winningLottoSection.classList.replace('show', 'hidden');
+  }
+
+  reloadView() {
+    this.$winningNumbersInputs.forEach((numberInput) => (numberInput.value = ''));
+    this.$bonusNumberInput.value = '';
+  }
+
+  disableResultButton() {
+    this.$resultButton.setAttribute('disabled', true);
```

Comment:
```suggestion
    this.$resultButton.disabled = true;
```

요거랑 setAttribute랑은 어떤 차이가 있을까요? 😄

Reply:
두 경우 모두 같은 결과를 내는 것 같은데 혹시 가독성에서 2번이 더 좋은것인가요??

Reply:
```js
/** Sets the value of element's first attribute whose qualified name is qualifiedName to value. */
    setAttribute(qualifiedName: string, value: string): void;
```


`setAttribute` 메소드를 호출해서 속성값을 할당하는 것과, 바로 속성값을 할당하는 것에도 차이는 있지만, (요건 너무 길어져서 생략)

`this.$resultButton.setAttribute('disabled', false);`

가 제대로 동작하는지 한번 확인해보실래요? 

참고: https://developer.mozilla.org/en-US/docs/Web/API/Element/setAttribute